packages/Spellcheck-Core.package/SpellcheckCorrectionUI.class/instance/selectedAlternativeIndex..st

```diff
@@ -1,8 +1,9 @@
 as yet unclassified
 selectedAlternativeIndex: anIndex
 	anIndex = 0 ifTrue: [^self].
-	inputText replaceFrom: currentKey to: (wrongWords at: currentKey) with: (self alternatives at: anIndex).
-	Transcript show: inputText.
-	self next.
-	"selectedIndex := anIndex."
-	"self changed: #selectedAlternativeIndex."
\ No newline at end of file
+	selectedIndex := anIndex.
+	self changed: #selectedAlternativeIndex.
+	workspace 
+		ifNotNil: [self replaceWorkspaceTextWithAlternative.] 
+		ifNil: [inputText replaceFrom: currentKey to: (wrongWords at: currentKey) with: (self alternatives at: anIndex).].
```

Comment:
I am not sure if we should restrict the tool to only be used in the workspace!? We could get rid of the case `ifNil`